patches/react-native-background-timer+2.4.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native-background-timer/android/build.gradle b/node_modules/react-native-background-timer/android/build.gradle
+index 85cda09..7661a6b 100644
+--- a/node_modules/react-native-background-timer/android/build.gradle
++++ b/node_modules/react-native-background-timer/android/build.gradle
+@@ -13,6 +13,8 @@ android {
+         versionCode 1
+         versionName "1.0"
+     }
++
++    namespace "com.ocetnik.timer"
```

Comment:
Rather than this, apply it on this fresh fork: https://github.com/jitsi/react-native-background-timer